tests/admin_scripts/tests.py

```diff
@@ -118,17 +119,26 @@ def run_test(self, args, settings_file=None, apps=None):
         test_environ['PYTHONPATH'] = os.pathsep.join(python_path)
         test_environ['PYTHONWARNINGS'] = ''
 
+        additional_kwargs = dict()
+        if umask and sys.hexversion >= 0x3090000:
+            # umask setting in subprocess.run() only supported from 3.9+
+            current_umask = os.umask(0)
+            os.umask(current_umask)
+            applied_umask = umask or current_umask
+            additional_kwargs['umask'] = applied_umask
```

Comment:
Do we need to check a `current_mask`? This code is reachable only when `umask` is set.
```suggestion
            kwargs['umask'] = umask
```

Reply:
Good point - unneeded leftover from a previous iteration.